### ISSUE TITLE
Height overlapping issue

### ISSUE THREAD

**Issue by squaredonut** [zippyui/react-datagrid#208]
So when I have the DataGrid on a page, that has a dynamically updating list of items. Once the items have been loaded, the last two rows seem to overlap on top of my contents. I have set an explicit height on the component.

It seems like it goes a bit beyond the height I set, so i tried increasing it, but it seems like a timing issue with calculating the height or what not.

**Comment by squaredonut** [comment]
![screen shot 2016-05-09 at 9 42 58 pm](https://cloud.githubusercontent.com/assets/484636/15112079/2d7d8fe8-162f-11e6-9dba-cab18c1d0de1.png)

This is what the overlapping looks like, and all the markup is:

`          <div>        
                <DataGrid idProperty='EntityId' dataSource={this.state.contacts} style={{height: 400, width: 700}} columns={[{name: 'DisplayName'}, {name:'Email'}]} />
                  <input type="text" placeholder="hello world" />
           </div>`

**Comment by chipilov** [comment]
I actually see a similar issue in v2.0.2. when adding a border around the table - the scrollable area of the grid overlaps the border of the table:
![table-border-overlap](https://cloud.githubusercontent.com/assets/389130/15140225/c8a0b478-16a2-11e6-974e-20fbc12ad9f0.png)

Here's the markup:

```
<DataGrid style={{ height: 60 }}
                  columns={[ { name: "name" }, { name: "capacity" } ]}
                  dataSource={this.state.tables}
                  idProperty="id"
                  showCellBorders={true}
                  withColumnMenu={false} />;
```

**Comment by chipilov** [comment]
I just compared my table and your example at <http://zippyui.com/react-datagrid/#/examples/custom-cell-borders> and in your example the top-level div (i.e. the one marked ```.react-datagrid```) has ```display: flex``` applied to it, while I don't see that in my table. Maybe something changed in v2.0.2 compared to the version from the example?

In any case, this is a workaround that seems to fix the issue (i.e. adding ```display: flex``` to the ```.react-datagrid``` class):
```
$table-border: 1px solid black;

.react-datagrid {
  border: $table-border;
  display: flex;

  .z-header-wrapper {
    background: none;
    border-bottom: $table-border;
  }
}
```

**Comment by squaredonut** [comment]
@chipilov  thanks for that, works like a treat. A question for you, do we have to set the border for the entire grid ourselves, because I thought it was a default style, but it does not appear that way, the examples all seem to have the black border.

**Comment by chipilov** [comment]
Yes, I think that's the case - I don't see any React props that might control the border and @radubrehar seems to imply that you need to tweak the CSS classes in the discussion for #188

**Comment by kevinrue** [comment]
Would be really awesome if @chipilov 's fix could be PR'ed into the default CSS, no?
Thanks for the fix anyway: works like a charm!
